### PR TITLE
Add okcoin

### PR DESCRIPTION
**Pull request by MikeHathaway**

### CODE REVIEW

File: bin/okcoin-market-maker-keeper [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+dir="$(dirname "$0")"/..
+source $dir/_virtualenv/bin/activate || exit
+export PYTHONPATH=$PYTHONPATH:$dir:$dir/lib/pymaker:$dir/lib/pyexchange:$dir/lib/ethgasstation-client:$dir/lib/gdax-client
+exec python3 -m market_maker_keeper.okcoin_market_maker_keeper $@
```
Comment (EdNoepel) [inline bin/okcoin-market-maker-keeper]: Recommend ending file with a newline.

Comment (MikeHathaway) [reply bin/okcoin-market-maker-keeper]: Added newline

**Comment by MikeHathaway** [comment]
> Is there an opportunity to subclass `OkcoinMarketMakerKeeper` from `OkexMarketMakerKeeper`? Admittedly, most of the keepers have a lot of duplicated code. But this seems like a shining opportunity for code reuse. I think you`d just need a new `**init**` method.

I'm not sure there's a way to do that without also modifying the OkexMarketMakerKeeper - there's a number of hardcoded arguments like `prog` that would need to be parameterized. This would be easily doable, but it would also require changes to production keepers. 

Given the similarities between all the keepers, would it instead be worthwhile to work on developing a common abstract class?

**Comment by MikeHathaway** [comment]
> Each subclass would need their own `__init__` method. @grandizzy introduced `PyexApi` after many `pyexchange` integrations were baked, so at least integrations which followed, and API updates, could make use of the interface. Perhaps for future integrations, we could develop such a base class.

I agree. Perhaps a Base class for CEX and a base for DEX?

**Pull request merged**